src/warnet/main.py

```diff
@@ -225,6 +231,37 @@ def auth(kube_config: str) -> None:
         )
 
 
+@cli.command()
+@click.argument("pod_name", type=str, default="")
+@click.option("--follow", "-f", is_flag=True, default=False, help="Follow logs")
+def logs(pod_name: str, follow: bool):
+    """Show the logs of a pod"""
+    namespace = get_default_namespace()
+    pods = run_command(f"kubectl get pods -n {namespace} -o json")
```

Comment:
if the user explicitly passes the name of the pod they would like logs for, seems better to directly try and get logs for that pod instead of allows asking the cluster for a list of pods. if we have 1,000 pods, this will make the log command slow every time, not just in the case where the user wants to use the selector menu.

I'd recommend if the user does not provide a pod name, it brings up the selector menu. if a user does provide a pod name, it tries to get the logs and if it fails tells the user the pod doesn't exist and prompts them to use the selector menu.

Reply:
@josibake I reworked the workflow based on your advice.